pkgs/applications/office/watson/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, pythonPackages }:
+
+pythonPackages.buildPythonApplication rec {
+  pname = "td-watson";
+  name = "${pname}-${version}";
+  version = "1.4.0";
+
+  src = pythonPackages.fetchPypi {
+    inherit version pname;
+    sha256 = "1py0g4990jmvq0dn7jasda7f10kzr41bix46hnbyc1rshjzc17hq";
+  };
+
+  # uses tox, test invocation fails
+  doCheck = false;
+  #checkPhase = ''
+  #  tox $out
```

Comment:
> I'd like to avoid that kind of work.

We all like to avoid work but that's what it takes to get a package in Nixpkgs.

Check what is inside the `tox.ini`. Likely you can just run `py.test`.